src/jdk.naming.rmi/share/classes/module-info.java

```diff
@@ -60,8 +60,9 @@
  * </ul>
  * <p> Downloading a factory class from a {@linkplain javax.naming.Reference#getFactoryClassLocation()
  * location} specified in the reference can be supported by a custom implementation of {@link
- * javax.naming.spi.ObjectFactoryBuilder}. Unless an {@link javax.naming.spi.ObjectFactoryBuilder
- * ObjectFactoryBuilder} is installed a {@link javax.naming.ConfigurationException} is thrown.
+ * javax.naming.spi.ObjectFactoryBuilder}. If a location is specified, then
+ * unless an {@link javax.naming.spi.ObjectFactoryBuilder} is installed a
```

Comment:
Minor: I think you need:
```
unless an {@link javax.naming.spi.ObjectFactoryBuilder ObjectFactoryBuilder}
```
to avoid the uncertainty about whether this would be rendered as "an javax...." or "an Object..."

Reply:
I've checked the generated javadoc, and the link is rendered as:
  "If a location is specified, then unless an _ObjectFactoryBuilder_"

Reply:
Good, thanks for double checking :-)